app/services/analytics_events.rb

```diff
@@ -779,26 +779,39 @@ def idv_intro_visit
   end
 
   # @param [Boolean] success
+  # @param [Hash] proofing_components User's current proofing components
   # Tracks the last step of IDV, indicates the user successfully prooved
   def idv_final(
     success:,
+    proofing_components:,
     **extra
   )
     track_event(
       'IdV: final resolution',
       success: success,
+      proofing_components: proofing_components,
```

Comment:
just confirming - the proofing components are the PROOFING_COMPONENT_KEYS correct?

Reply:
Correct. Technically the analytics method itself doesn't impose any requirements about the shape or type of the `proofing_components` value, but with how it's provided in the `AnalyticsDecorator` class, we'd expect a hash with those keys.